### DISCUSSION TITLE
Shared memory between host and guests?

### DISCUSSION THREAD

**Discussion by PhilippGrulich** [cloud-hypervisor/cloud-hypervisor#2530] [Q&A]
Hi,

sorry for the maybe obvious question.
But after reading the documentation I was wondering if it is possible to share a memory region between host and guest with cloud-hypervisor.

Thanks,
Philipp

**Comment by rbradford** [comment]
@PhilippGrulich Sharing memory between the guest and a separate process on the host is the mechanism `vhost-user` provides.
